tokens/russian.yml

```diff
@@ -0,0 +1,25 @@
+################
+# A token might be a variable or function name (AKA an identifier), an operator or a number.
+# USE C++ match_regex SYNTAX TO WRITE NEW TOKENS
+################
+^[0-9]+$: INTEGER
+^\+$: ADD
+^-$: SUB
+^\*$: MULT
+^/$: DIVIDE
+^;$: SEMICOLON
+^\($: OPEN_PARENTHESYS
+^\)$: CLOSE_PARENTHESYS
+^==$: EQUAL_TO
+^напечатать$: OUTPUT
+^ $: SPACE
+^\н$: NEW_LINE
```

Comment:
Can you change this NEW_LINE to `^\n$`? This should match ASCII new line.

Reply:
just wanted to double check before I make changes. H in Cyrillic is equivalent to N in roman alphabet. should i still make that change?